@market/hooks/useAskHelper.ts

```diff
@@ -31,15 +88,9 @@ export const useAskHelper = ({ ask }: AskHelperProps) => {
   const hasAsk = useMemo(() => ask !== undefined, [ask])
   const hasRelevantAsk = useMemo(() => hasAsk && isActiveAsk, [hasAsk, isActiveAsk])
 
-  const rawAskAmount = useMemo(() => ask?.amount?.amount.raw.toString(), [ask])
-  const displayAskAmount = useMemo(() => ask?.amount?.amount.value.toString(), [ask])
-  const usdAskAmount = useMemo(
-    () =>
-      ask?.amount?.usd?.value
-        ? numberFormatterUSDC(roundTwoDecimals(ask.amount.usd.value))
-        : '...',
-    [ask]
-  )
+  const rawAskAmount = useMemo(() => ask?.price?.nativePrice.raw, [ask])
+  const displayAskAmount = useMemo(() => ask?.price?.nativePrice.value, [ask])
+  const usdAskAmount = ask?.price.usdPrice
```

Comment:
Looks like the formatting has been stripped out here. Are we sure output would be correct?